### PR TITLE
Gp 2023.04.29

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt [hunk 1]
```diff
@@ -474,25 +474,30 @@ The earth's surface has been stored away,
...
+The giant of light rises from the ground.
 }
 [sha:c7d778846bb42775fe3ae29c24303bd1cfcb9505]{
 -- Page 30, Offset 39429.
 -- ソレは真祖という<規|カ><格|ラ>を見失い、<顕現|そんざい>方法が暴走しただけの力の具現であり―――
 // Maybe this should link with 39399?
-It is no more than an incarnation of power whose means of <manifestation|e x i s t e n c e   > has gone out of control,
- having lost its True Ancestor <coding|s h e l l   >.
+// Also maybe just reword this guy a bit
+It is no more than an incarnation of power whose means of <manifestation|e x i s t e n c e     > have gone out of control,
```
Comment (yk1512) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt:484]: With the loss of its True Ancestor shell, it is no more than an incarnation of power, whose means...

File: script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt [hunk 2]
```diff
@@ -517,6 +522,6 @@ Why wouldn't she?
 [sha:4a2db60cc895f54af7e6d74ad6f9bbf5c1c11153]{
 -- Page 32, Offset 39435.
 -- それは皮肉にもシエル自身が口にした、絶対的な事実だった。
-As ironic as it is,
- that is an ironclad doctrine to which Ciel herself had subscribed.
+// Referencing 39321
+Ironically, that is the undeniable fact that Ciel herself had stated earlier.
```
Comment (yk1512) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt:526]: preference to change it to 'Ironically, it is an undeniable fact'

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt [hunk 3]
```diff
@@ -551,7 +566,7 @@ It gusts around the Luminary like a tornado,
 [sha:f4b56cf0aab3bef9bda8a57edf6701b762233b9e]{
 -- Page 163, Offset 39883.
 -- 後退を余儀なくされる。
-I'm forced to retreat.
+I'm rced to retreat.
```
Comment (yk1512) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:569]: oopsy

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt [hunk 4]
```diff
@@ -220,7 +220,7 @@ But first, I've got to get there―――!
 [sha:1edccbb2b45ad3111b03929319c9ec772a159e86]{
 -- Page 143, Offset 39824.
 -- 何度目かの大気の爪を突破する。
-I slice my way through yet another of the atmospheric claws.
+I slice my way through yet another of the tempestuous claws.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:223]: can just replace all the 大気の爪 in this gpr with "claw/s of wind"

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt [hunk 5]
```diff
@@ -296,18 +296,22 @@ I don't even need to take advantage of my enhanced vision.
 -- Page 147, Offset 39837.
 -- いま、その横を通り過ぎようと躍起になっている俺に見せつけるように、その惨劇は行われていた。
 Just as I'm about to pass her,
- I catch sight of a cruel display seemingly put on for my benefit.
+ I catch sight of a cruel display seemingly put on for my 'enjoyment'.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:299]: I think we need a pronoun here
>...cruel display she has seemingly put on for my 'enjoyment'.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt [hunk 6]
```diff
@@ -387,12 +391,17 @@ That thing doesn't feed on protein,
 [sha:dc4be5129368a9a8682f2c09e8fe087037c9afac]{
 -- Page 153, Offset 39853.
 -- 「“もう色々びしょびしょね。
+// What did she mean by this
+%{i}
 "Looks like you're already wrung dry, aren't you?
+%{/i}
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:397]: As I said, let's just keep it fairly literal.
I haven't really found anyone else who can divine the purpose here.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt [hunk 7]
```diff
@@ -659,7 +674,7 @@ Even my millennia of love may fade at such a sight."
 -- Page 169, Offset 39900.
 -- 『出力はあがろうと、欠点が多くなればそれは弱体化だよ。
 %{i}
-"Even if the output power rises,
+"Even if her power is magnified,
  if the number of flaws increases,
  then she has weakened.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:679]: Still don't fully jive with this sentence.
I think I suggested more extensive changes, but regardless.
>Even if her power is magnified,
>when her flaws increase with it, 
>that would make her weaker.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt [hunk 8]
```diff
@@ -675,7 +690,7 @@ I am disappointed in Elesia.
 -- Page 169, Offset 39902.
 -- 大魔術だけで十分だったものを、必要以上に壊すとは。
 %{i}
-Even though her Grand Magecraft sufficed against Arcueid,
+Even though her grand magecraft sufficed against Arcueid,
  she destroyed her far more than was necessary.
 %{/i}
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:695]: Ah, I knew there was a problem here.
Roa calls her "Princess".
>Even though her grand magecraft would have sufficed, 
she destroyed the Princess far more than was necessary.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt [hunk 9]
```diff
@@ -844,32 +872,45 @@ Okay, hold on now...
...
+%{/i}
 }
 [sha:8e864146f7c1249a6ece72ad9d7301bfac0786fa]{
 -- Page 182, Offset 39936. Mods: @n.
 -- 僕は甘い仕草で初対面の女を解体する事しかできない、
-%{i}Ohh, look at me dismembering a woman I've never seen before all sensually%{/i}.
+// Maybe rework the 甘い仕草?
+%{i}
+Ohh, look at me dismembering a woman I've never seen before all sensually.
+%{/i}
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:906]: Something like this maybe?
>Ohh, look at me, acting all charming by dismembering a woman never seen before.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt [hunk 10]
```diff
@@ -176,37 +175,51 @@ Not only does it fail to penetrate her skin,
...
+%{i}
+Did you not even realize that %{/i}obvious%{i} fact?"
+%{/i}
 }
 [sha:6c567d3539d424a006b9b8047a4acb5f8e5f9223]{
 -- Page 208, Offset 40021.
 -- 『“おつかれさま。
+%{i}
 "You never fail to amaze.
+%{/i}
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt:201]: This one is a bit weird, but I'm not sure how to render it

File: script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt [hunk 11]
```diff
@@ -337,13 +337,13 @@ Her wealth of knowledge allows her to correctly intuit what the phenomenon befor
...
+Even though Ciel can sense the danger,
  her limbs are frozen in place.
 }
 [sha:5d245c13ae04f79bc66b0878e3802291c80fa27d]{
 -- Page 21, Offset 39406.
 -- それがこの現象の持つ魔眼―――その視界に入った時点で知性あるものは喪心する―――である事を、シエルは誰に教えられる事なく理解した。
-Ciel can tell that it's the result of this phenomenon's Mystic Eyes――
+She can tell that it's the result of this phenomenon's Mystic Eyes――
 capable of paralyzing any intelligent beings that come within their purview――
 without having to be told.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt:348]: > She understands instinctively that it's the result of this phenomenon's Mystic Eyes――
> eyes capable of paralyzing any intelligent beings that come within their purview.

?

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt (hunk 4 above)
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:224]: Claws of wind or air? I think we were using air by the time we got to 3b

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt (hunk 6 above)
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:399]: > Just look at what a mess you already are.

?

File: script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt (hunk 11 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt:348]: >She **instinctively** understands that it's the result of this phenomenon's Mystic Eyes――
eyes capable of paralyzing any intelligent beings that come within their purview.

small change, but seems good

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt (hunk 4 above)
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:224]: air, sure

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt (hunk 6 above)
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:399]: Sure, I guess

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt (hunk 10 above)
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt:201]: I think this is a sort of sarcastic "well done", but aside from literally just that, I don't know if there's a better way of rendering it

Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt:201]: "At least you tried." ?

Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt:201]: Let's go

Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt:201]: Is that a "sounds good, ship it"?

**Pull request merged**